### DISCUSSION TITLE
What's the difference between the getter in petite-vue and regular computed property in regular Vue?

### DISCUSSION THREAD

**Discussion by fnick851** [vuejs/petite-vue#217] [Q&A]
```js
createApp({
  // exposed to all expressions
  count: 0,
  // getters
  get plusOne() {
    return this.count + 1
  },
  // methods
  increment() {
    this.count++
  }
}).mount()
```

It seems like the getter is just a computed property in effect.

**Comment by halivert** [comment]
`get` properties are language native, but computed properties are Vue specific.
It depends how Vue computed properties are implemented, but I think `get` is more performant.
